src/Command/CommandInterface.php

```diff
@@ -138,8 +138,8 @@ public function addPrimaryKey(string $table, string $name, array|string $columns
     /**
      * Creates an SQL command for changing the definition of a column.
      *
-     * @param string $table The table whose column is to be changed.
-     * @param string $column The name of the column to be changed.
+     * @param string $table The table whose column is to change.
+     * @param string $column The name of the column to change.
```

Comment:
as far as I see, original is correct.  https://www.deepl.com/translator#de/en/Die%20Tabelle%2C%20dessen%20Spalte%20zu%20%C3%A4ndern%20ist.

Reply:
It is as correct as new one but new one isn't passive voice.